scripts/create_gitlab_snapshot.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import logging
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+from azul.terraform import (
+    _adjust_name_tag,
+    _tags,
+)
+
+logger = logging.getLogger(__name__)
+
+client = aws.client('ec2')
+sts_client = aws.client('sts')
```

Comment:
Not at top level.

Other scripts handle these as properties of a `Main` class.

Reply:
opted to use the aws.sts property, added aws.ec2 property.